agreement_legal/data/agreement_stage.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
```

Comment:
Add **encoding="utf-8"** in this line like <?xml version="1.0" encoding="utf-8"? >